includes/cli/class-woocommerce-subscriptions.php

```diff
@@ -162,20 +170,21 @@ public function migrate_expired_subscriptions( $args, $assoc_args ) {
 							}
 						}
 						++$scheduled;
-					} else {
-						// Otherwise, if the retry date is past the on-hold duration, update the subscription status to expired.
-						if ( self::$verbose ) {
-							WP_CLI::line( 'Updating subscription status to expired...' );
-						}
-						if ( self::$live ) {
-							$subscription->update_status( 'expired', __( 'Subscription status updated by Newspack CLI command.', 'newspack-plugin' ) );
-							$subscription->set_end_date( $retry_date );
-							$subscription->update_meta_data( '_newspack_cli_status_updated', true );
-							$subscription->save();
-						}
-						++$updated;
 					}
 				}
+				if ( should_expire ) {
```

Comment:
typo, missing `$`

Reply:
fixed in https://github.com/Automattic/newspack-plugin/pull/3663/commits/16605d5b3f8bace16d8a8ada00214611512644ab